### DISCUSSION TITLE
Autosquash can't squash commits that modify non-formula files

### DISCUSSION THREAD

**Discussion by dpo** [Homebrew/discussions#3033] [Tap maintenance and Homebrew development] [answered]
Hi,

I help maintain the tap https://github.com/coin-or-tools/homebrew-coinor and submitted a PR that only modifies the version of a formula in order to rebuild the bottles. The brew test-bot jobs succeed, but when I label the PR with "pr-pull", the job fails with the message
```
Error: Autosquash can't squash commits that modify non-formula files.
```
https://github.com/coin-or-tools/homebrew-coinor/runs/5328041326?check_suite_focus=true#step:4:20

However, the PR only modifies a formula file: https://github.com/coin-or-tools/homebrew-coinor/pull/81/files

Any ideas what could be going wrong here?

Thanks in advance!

**Comment by Bo98** [comment, marked as answer]
This appears to be a bug - feel free to open an issue at Homebrew/brew. Looks like it is hardcoded to only check the `Formula` subdirectory: https://github.com/Homebrew/brew/blob/b4244bb1608ccf71ecc7cd3b61e014e88009af44/Library/Homebrew/dev-cmd/pr-pull.rb#L227

(A workaround would be to use `--no-autosquash`, though this will of course disable commit squashing should you rely on that.)

**Reply by dpo** [reply]
Thanks! I opened https://github.com/Homebrew/brew/issues/12939.
